src/App.js

```diff
@@ -3,6 +3,12 @@ import ProbList from './ProbList';
 import Prob from './Prob'
 import { BrowserRouter, Route } from 'react-router-dom';
 import Navbar from './Components/Main/Navbar';
+
+/*
```

Comment:
Please remove this comment. It is un necessary

Reply:
Sure Why not